setup.cfg

```diff
@@ -44,8 +44,8 @@ filterwarnings =
     # Source: test_sct_register_multimodal_mto_image_data --> `algorithms.py :: vxm.networks.VxmDense()`
     ignore:.*in an upcoming release, it will be required to pass the indexing argument.*:UserWarning:torch.*:
     # This warning is due to a deprecation for a call that is still used by many upstream packages. In our case:
-    # - wandb: https://github.com/wandb/wandb/issues/6711
-    ignore:pkg_resources is deprecated as an API:DeprecationWarning
+    # - monai: https://github.com/Project-MONAI/MONAI/issues/7559
```

Comment:
Thanks for the attempted cleanup!! Even though the result is just a comment change, it's very important for future understandability. Plus, it led you to report it upstream, which is great open source citizenship.